### PR TITLE
Add TiSpark blog

### PR DESCRIPTION
**Pull request by yikeke**

### CODE REVIEW

File: batch-processing-massive-data-much-quicker-with-tispark.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+author: ['Zhexuan Yang']
+date: 2020-12-22
+summary: A senior solution architect at PingCAP explains why TiSpark (TiDB + Apache Spark) is better than the traditional batch processing solution, how you can benefit from TiSpark, and how it works.
+tags: ['TiSpark']
+categories: ['Engineering']
+image: /images/blog/batch-processing-massive-data-much-quicker-with-tispark.jpg
+aliases: ['/blog/batch-processing-massive-data-much-quicker-with-tispark']
+---
+
+![Batch Processing Massive Data Much Quicker with TiSpark](media/batch-processing-massive-data-much-quicker-with-tispark.jpg)
```
Comment (yikeke) [inline batch-processing-massive-data-much-quicker-with-tispark.md:12]: I‘ll add the banner later.

**Comment by yikeke** [comment]
PTAL @CaitinChen

File: batch-processing-massive-data-much-quicker-with-tispark.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
+---
+title: Batch Processing Massive Data Much Quicker with TiSpark
+author: ['Zhexuan Yang']
+date: 2020-12-22
+summary: A senior solution architect at PingCAP explains why TiSpark (TiDB + Apache Spark) is better than the traditional batch processing solution, how you can benefit from TiSpark, and how it works.
+tags: ['TiSpark']
+categories: ['Engineering']
+image: /images/blog/batch-processing-massive-data-much-quicker-with-tispark.jpg
+aliases: ['/blog/batch-processing-massive-data-much-quicker-with-tispark']
```
Comment (CaitinChen) [inline batch-processing-massive-data-much-quicker-with-tispark.md:9]: Why do you add an alias?

File: batch-processing-massive-data-much-quicker-with-tispark.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+---
+
+![Batch Processing Massive Data Much Quicker with TiSpark](media/batch-processing-massive-data-much-quicker-with-tispark.jpg)
+
+TiSpark is an Apache Spark plugin that works with the TiDB platform to answer complex online analytical processing (OLAP) queries. It is widely used by our customers to batch process their massive data and discover analytical insights.
```
Comment (CaitinChen) [inline batch-processing-massive-data-much-quicker-with-tispark.md:14]: Please add a link for "TiSpark" and "TiDB."
I also suggest linking "customers" to https://en.pingcap.com/case-studies.

File: batch-processing-massive-data-much-quicker-with-tispark.md (hunk 2 above)
Comment (yikeke) [reply batch-processing-massive-data-much-quicker-with-tispark.md]: I copied it from another blog. Can you add the user guide for the aliases in our blog writing tutorial?

File: batch-processing-massive-data-much-quicker-with-tispark.md (hunk 3 above)
Comment (yikeke) [reply batch-processing-massive-data-much-quicker-with-tispark.md]: done

**Comment by CaitinChen** [comment]
For SEO purposes, you'd better add image captions to all images except the banner.

**Review by CaitinChen — APPROVED** [review]

**Pull request merged**